src/components/ProjectSettingsTabs/VariablesTab/getFormSchema.ts

```diff
@@ -84,6 +88,11 @@ export const getFormSchema = (
             "Private variables have redacted values on the Project Page and the API and cannot be updated.",
           "ui:data-cy": "var-private-input",
         },
+        isAdminOnly: {
+          "ui:tooltipDescription":
+            "Admin only variables are only accessible by project admins.",
```

Comment:
"Accessible" isn't super helpful, since technically all variables can only be viewed by project admins. Maybe "can only be used by"

Reply:
done